src/smtlib/proc.rs

```diff
@@ -550,6 +575,11 @@ impl SmtProc {
             let n = self.stdout.read_line(&mut buf)?;
             if n == 0 {
                 self.check_killed()?;
+                // TODO(tchajed): this is just from observing behavior, I'm not sure
+                // why check_killed doesn't work here
+                if buf.is_empty() {
+                    self.assert_killed()?;
```

Comment:
This comment is the same as above, but the body of the if statement is `assert_killed` here rather than `check_killed` in the previous instance of this comment. Is that correct?